src/main/com/algos/src/maths/AbsoluteMax.java

```diff
@@ -13,12 +13,12 @@ public static int getMaxValue(int... numbers) {
         if (numbers == null || numbers.length == 0) {
             throw new IllegalArgumentException("Numbers array cannot be empty or null");
         }
-        int absMax = numbers[0];
+        int absMax = 0;  
         for (int i = 1; i < numbers.length; i++) {
-            if (Math.abs(numbers[i]) > Math.abs(absMax)) {
+            if (numbers[i] > absMax) {
                 absMax = numbers[i];
             }
         }
-        return absMax;
+        return Math.abs(absMax);
```

Comment:
**Performance Issue**: The modification of the initial value of absMax and the comparison logic in the loop does not correctly find the absolute maximum value. It incorrectly assumes positive numbers are always greater and fails to consider negative numbers with large absolute values. <br> **Fix**: Initialize absMax with the first element of the numbers array to handle both positive and negative numbers correctly. Use Math.abs in the comparison to ensure the absolute maximum value is found. <br> **Code Suggestion**: 
 ```
 -        int absMax = 0;
 +        int absMax = numbers[0];
 -            if (numbers[i] > absMax) {
 +            if (Math.abs(numbers[i]) > Math.abs(absMax)) {
 ```